de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ExporterImporterWizardTest.java

```diff
@@ -74,15 +65,9 @@ public void testExcelExportImportFEA() {
 		
 		// Open the import menu, workaround to shells sometimes not being valid anymore after closing a wizard
 		// See https://wiki.eclipse.org/SWTBot/Troubleshooting#WidgetNotFoundException_when_stepping_through_SWTBot_test_in_Eclipse_debugger for details
```

Comment:
Is this comment stil relevant, should we move it into the function then?

Reply:
Removed the open import menu comment.
Otherwise the comment is still valid.